README.md

```diff
@@ -88,6 +88,26 @@ use { "johmsalas/text-case.nvim",
   -- of the keymappings, e.g. `gau ` executes the `current_word` method with `to_upper_case`
   -- and `gaou` executes the `operator` method with `to_upper_case`.
   prefix = "ga",
+  -- By default, all methods are enabled. If you set this option with some methods omitted,
+  -- these methods will not be registered in the default keymappings. The methods will still
+  -- be accessible when calling the exact lua function e.g.:
+  -- "<CMD>lua require('textcase').current_word('to_snake_case')<CR>"
+  enabled_methods = {
```

Comment:
Should enabled_methods also be used for Telescope? I'm wondering whether the enabled methods table should be independent for customization

Reply:
My plan was to add it to Telescope as well after merging. But for other things like Subs not really. Because it's only about changing the "interface" of whichkey and Telescope.

Reply:
What do you mean by independent?

Reply:
Having 2 different properties, to customize the selected options.
I think enabled_methods is fine, 2 options would be overwhelming

Reply:
Yeah, it might be too much.